src/OrleansProviders/Streams/Common/Monitors/DefaultBlockPoolMonitor.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orleans.Providers.Streams.Common
+{
+    public class DefaultBlockPoolMonitor : IBlockPoolMonitor
+    {
```

Comment:
DefaultBlockPoolMonitor which is used in MemoryStreamProvider and GeneratorStreamProvider as a default monitor option